src/jupyter_contrib_nbextensions/nbextensions/runtools/runtools.yaml

```diff
@@ -4,4 +4,61 @@ Description: Runtools provide a number of additional functions for working with
 Link: readme.md
 Icon: icon.png
 Main: main.js
-Compatibility: 4.x
+Compatibility: 5.x
+Parameters:
+- name: runtools.show_cellstate
+  description: |
+    Display indicator for individual cell state (locked, modified, rund)
+  default: false
+  input_type: checkbox
+- name: runtools.run_cells_above
+  description: run cells above
+  input_type: hotkey
+  default: Alt-a
+- name: runtools.run_cells_below
+  description: run cells below
+  input_type: hotkey
+  default: Alt-b
+- name: runtools.toggle_marker
+  description: Toggle marker
+  input_type: hotkey
+  default: Alt-t
+- name: runtools.mark_all_codecells
+  description: Mark all codecells
+  input_type: hotkey
+  default: Alt-m
+- name: runtools.unmark_all_codecells
+  description: Unmark all codecells
+  input_type: hotkey
+  default: Alt-u
+- name: runtools.run_marked_cells
+  description: run marked cells
+  input_type: hotkey
+  default: Alt-r
+- name: runtools.stop_execution
+  description: stop execution
+  input_type: hotkey
+  default: Ctrl-c
+- name: runtools.run_all_cells
+  description: run all cells
+  input_type: hotkey
+  default: Alt-x
+- name: runtools.run_alls_cells_ignore_errors
+  description: run all cells - ignore errors
+  input_type: hotkey
+  default: Alt-f
+- name: runtools.marked_color
+  description: |
+    Color for marking a codecell
+  default: '#0f0'
```

Comment:
I have a feeling that the configurator fails to load correctly 3-character colour codes, so it's probably best to use the full 6-char code here